pyproject.toml

```diff
@@ -163,25 +163,24 @@ ignore = [
     "PLR0915",
     "PLR1704",
     "PLR2004",
-    "PTH100",
-    "PTH101",
-    "PTH102",
+#    "PTH100",
```

Comment:
The merge conflict is maybe because of the difference in code of the pyproject.toml file, I exactly don't know how to resolve that. I think that file is constantly getting changed so causing the issue.

Reply:
let me try to fix it

Reply:
OK I think this is fixed: make sure you `git pull` before making more changes.